src/bidiMapper/modules/network/NetworkRequest.ts

```diff
@@ -237,14 +237,30 @@ export class NetworkRequest {
     return bodySize;
   }
 
-  get #context() {
-    return (
+  get #context(): string | null {
```

Comment:
let's add a unit test by checking an emitted event for this scenario as well? src/bidiMapper/modules/network/NetworkStorage.spec.ts